src/chronify/schema_manager.py

```diff
@@ -0,0 +1,94 @@
+import json
+
+from loguru import logger
+from sqlalchemy import (
+    Column,
+    Connection,
+    Engine,
+    MetaData,
+    String,
+    Table,
+    delete,
+    insert,
+    select,
+)
+
+from chronify.exceptions import (
+    TableNotStored,
+)
+from chronify.models import (
+    TableSchema,
+)
+
+
+class SchemaManager:
+    """Manages schemas for the Store. Provides a cache to avoid repeated database reads."""
```

Comment:
Forcing a user query to read the schema from the db every time was adding latency.